### ISSUE TITLE
What's kind of uri address have got this node-stun server on a local desktop?

### ISSUE THREAD

**Issue by Globik** [enobufs/stun#9]
Hi, I'm wondering about the uri address of this node-stun. How it should be looking? Where the place for ice config options.

**Comment by enobufs** [comment]
This is a partial implementation of old RFC 3489, it waits on a specific set of IP address and port number. How it maps to URI and where `ICE config` is is out of scope of this implementation.
